tests/QueueTest.php

```diff
@@ -146,17 +147,22 @@ public function it_posts_the_task_the_correct_queue()
         // Arrange
         CloudTasksApi::fake();
 
+        $closure = fn () => 'closure job';
+        $closureDisplayName = CallQueuedClosure::create($closure)->displayName();
```

Comment:
Display name of a closure job looks like `Closure (QueueTest.php:150)`. Added this line to keep the ability to assert `displayName` in `CloudTasksApi::assertTaskCreated()`.